pkg/pillar/zboot/zboot.go

```diff
@@ -428,35 +430,21 @@ func getVersion(log *base.LogObject, part string, verFilename string, inContaine
 		mountFlags := MountFlagRDONLY
 		err = zbootMount(devname, target, "squashfs", mountFlags, "")
 		if err != nil {
-			log.Errorf("Mount of %s failed: %s\n", devname, err)
-			return ""
+			errStr := fmt.Sprintf("Mount of %s failed: %s", devname, err)
+			log.Errorln(errStr)
+			return "", errors.New(errStr)
 		}
 		defer syscall.Unmount(target, 0)
-		var filename string
-		if inContainer {
-			filename = fmt.Sprintf("%s/%s/%s",
-				target, otherPrefix, verFilename)
-		} else {
-			filename = fmt.Sprintf("%s/%s",
-				target, verFilename)
-		}
+		filename := fmt.Sprintf("%s/%s",
+			target, verFilename)
 		version, err := ioutil.ReadFile(filename)
 		if err != nil {
 			log.Warn(err)
-			if !inContainer {
-				return ""
-			}
-			filename := fmt.Sprintf("%s/%s/%s",
-				target, otherPrefixOld, verFilename)
```

Comment:
Please remove otherPrefixOld and associated comments as well.

Reply:
ok, will do.

Reply:
@zed-rishabh didn't see an update.